src/components/select/template.test.js

```diff
@@ -135,6 +135,19 @@ describe('Select', () => {
       expect($firstItem.attr('disabled')).toBeTruthy()
     })
 
+    it('renders with aria-describedby', () => {
+      const describedById = 'parent-fieldset-hint-id'
+
+      const $ = render('select', {
+        describedBy: describedById
+      })
+
+      const $component = $('.govuk-select')
+
+      expect($component.attr('aria-describedby'))
```

Comment:
Again non-blocking, but I'd just chain `$('govuk-select')`

Reply:
@36degrees Mind if we keep it but I'll group it better?

It's done like this throughout a lot of tests:

```js
const $component = $('.govuk-select')
expect($component.attr('aria-describedby')).toMatch(describedById)
```

Reply:
So if the `expect()` comes after the `$component` declaration it's now as above.
i.e. Two lines, variable first, with the assertion all on one line.

If it's following other variable declarations like `errorMessageId` and `hintId` it's written like so:

```js
expect($component.attr('aria-describedby'))
  .toMatch(errorMessageId)
```

I think I understand your code style now 😁

Reply:
TBH, I don't think we're terribly good as being consistent on this, so this is more my personal preference than anything 😬

Happy for you to leave it as it is and we'll try and firm up our style a little and go through and tidy up… one day…